spec/models/article_spec.rb

```diff
@@ -107,6 +107,46 @@ def build_and_validate_article(*args)
       end
     end
 
+    describe "#validate_tag" do
+      # rubocop:disable RSpec/VerifiedDoubles
```

Comment:
Ah, because `TagList` implements `add` and `remove` but our model also needs to call `Array#sort`, we can't use a verifying double. :+1: